src/metabase/models/action.clj

```diff
@@ -229,12 +223,25 @@
 
                                     :always seq))))))
 
+(defn select-action
+  "Selects an Action and fills in the subtype data and implicit parameters.
+   `options` is passed to `db/select-one` `& options` arg."
+  [& options]
+  (first (apply select-actions nil options)))
+
+(defn select-action-without-implicit-params
+  "Selects an Action and fills in the subtype data. Doesn't include generated parameters for implicit actions,
```

Comment:
Should we include in the docstring that if we need actions with implicit params, you should use `select-action(s)`?

Reply:
👍